docs/sources/tempo/traceql/_index.md

```diff
@@ -208,20 +208,44 @@ The second expression returns no traces because it's impossible for a single spa
 
 ### Structural
 
-These spanset operators look at the structure of a trace and the relationship between the spans.
+These spanset operators look at the structure of a trace and the relationship between the spans. Structural operators ALWAYS return
+matches from the right hand side of the operator.
 
 - `{condA} >> {condB}` - The descendant operator (`>>`) looks for spans matching `{condB}` that are descendants of a span matching `{condA}`
 - `{condA} << {condB}` - The ancestor operator (`<<`) looks for spans matching `{condB}` that are ancestor of a span matching `{condA}`
 - `{condA} > {condB}` - The child operator (`>`) looks for spans matching `{condB}` that are direct child spans of a parent matching `{condA}`
 - `{condA} < {condB}` - The parent operator (`<`) looks for spans matching `{condB}` that are direct parent spans of a child matching `{condA}`
-- `{condA} ~ {condB}` - The sibling operator (`~`) checks that spans matching `{condA}` and `{condB}` are siblings of the same parent span.
+- `{condA} ~ {condB}` - The sibling operator (`~`) looks that spans matching `{condB}` that have at least one sibling matching `{condA}`.
 
 For example, to find a trace where a specific HTTP API interacted with a specific database:
 
 ```
 { span.http.url = "/path/of/api" } >> { span.db.name = "db-shard-001" }
 ```
 
+### Experimental Structural
+
+These spanset operators look at the structure of a trace and the relationship between the spans. They are marked experimental b/c they
+are known to sometimes return false positives. However, they can be very useful and we encourage users to try them and give feedback.
```

Comment:
We say that these operators can be useful. Do we need to elaborate more on that or are the provided examples enough? 
```suggestion
These spanset operators look at the structure of a trace and the relationship between the spans. They are marked experimental because they
are known to sometimes return false positives. However, they can be very useful (see examples below). We encourage users to try them and give feedback.
```